apps/cookbook/src/app/examples/slides-example/slides-example.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'cookbook-slides-example',
+  templateUrl: './slides-example.component.html',
+})
+export class SlidesExampleComponent {
+  constructor() {}
+
+  slidesOptions = {
+    spaceBetween: 5,
+    centeredSlides: true,
+    slidesPerView: 1.15,
+    breakpoints: {
+      721: {
+        slidesPerView: 2.15,
+        centeredSlides: false,
+      },
+    },
+  };
+
+  slides = [1, 2, 3, 4, 5];
+
+  hasPadding = true;
+  showSize = true;
+  hasHeader = true;
+  title = 'Test Title';
+  subtitle = 'Test Subtitle';
+  hasHeaderFooterBgColor = true;
+  hasFooter = true;
+
+  getDataFromActiveSlide(e: any) {
```

Comment:
What are the usecase for this?

Reply:
Have added a comment - its the output callback